microprofile/WFLY-13023_upgrade_microprofile-config_1.4.adoc

```diff
@@ -0,0 +1,58 @@
+= Upgrade MicroProfile Config to 1.4
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,config,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 3.3 provides a minor upgrade for https://github.com/eclipse/microprofile-config/releases/tag/1.4[MicroProfile Config 1.4].
+
+This component upgrade also covers the upgrade of smallrye-config that implements MicroProfile Config 1.4.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13023[WFLY-13023]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1427[EAP7-1427]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:mkopecky@redhat.com[Marek Kopecky]
+
+=== Testing By
+
+[X] QE
+
+== Requirements
+
+* Upgrade MicroProfile Config to 1.4
+* Upgrade smallrye-config to the release that implements MicroProfile Config 1.4
+* Run any MicroProfile Health test in WildFly integration test suite
+* Run the Eclipse MicroProfile Health TCK without failures
+
+=== Non-Requirements
+
+== Implementation Plan
+
+== Test Plan
```

Comment:
@jmesnil What is important to have test coverage in your PoV? Currently, this section doesn't contain anything about new tests.

Reply:
I mentioned that spec changes are covered by the MP Config TCK and do not require new tests in WildFly